sources/packages/backend/apps/load-test-gateway/src/main.ts

```diff
@@ -0,0 +1,14 @@
+import "../../../env-setup";
+import { NestFactory } from "@nestjs/core";
+import { LoadTestModule } from "./load-test.module";
+import { ConfigService } from "@sims/utilities/config";
+import { KeycloakConfig } from "@sims/auth/config";
+
+async function bootstrap() {
+  await KeycloakConfig.load();
+  const app = await NestFactory.create(LoadTestModule);
+  app.setGlobalPrefix("load-test");
+  const config = app.get<ConfigService>(ConfigService);
+  await app.listen(config.loadTestAPIPort);
```

Comment:
The `setGlobalPipes` are not set, are the default values enough?
I am keeping in mind that even this API not being supposed to be deployed to PROD it will be public available.